test/fixtures/no_preprocessor/input.ino

```diff
@@ -0,0 +1,13 @@
+// This should not be preprocessed
+
+#pragma PARTICLE_NO_PREPROCESSOR
+
+#include "Particle.h"
+
+int handler(int foo) {
```

Comment:
This is what this test is doing. This file is the input. Below is the output. The output must match exactly. So since it does not include #line or #include "application.h" it shows the preprocessor not running.